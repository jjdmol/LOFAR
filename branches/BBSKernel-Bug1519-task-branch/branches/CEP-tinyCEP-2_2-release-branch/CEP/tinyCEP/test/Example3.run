#!/bin/sh

# Run the Example3 test program by starting it twice.
# Once as sender and once as receiver.

./Example3 -s > Example3_tmp.outs &
./Example3 -r > Example3_tmp.outr

# If something went wrong, kill the background process (if it's still running)
# and exit.
STATUS=$?
if [ $STATUS != 0 ]; then
  if kill -0 $! 2>/dev/null; then
    kill -9 $! 2>/dev/null
  fi
  exit $STATUS
fi

cat Example3_tmp.outs Example3_tmp.outr
