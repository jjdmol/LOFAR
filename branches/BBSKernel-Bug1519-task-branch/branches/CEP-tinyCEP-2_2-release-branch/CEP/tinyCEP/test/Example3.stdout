Sender dump:
InHolder(0) : (4,3)
OutHolder(0): (0,0)
Sender dump:
InHolder(0) : (4,3)
OutHolder(0): (4,3)
Receiver dump:
InHolder(0) : (0,0)
OutHolder(0): (0,0)
Receiver dump:
InHolder(0) : (4,3)
OutHolder(0): (4,3)
